protocol.html

```diff
@@ -682,6 +686,35 @@ <h3 property="schema:name"><span>Note</span>: Storage Owner and URI Ownership</h
                 </div>
               </div>
 
+              <section id="iris" inlist="" rel="schema:hasPart" resource="#iris">
+                <h3 property="schema:name">Internationalized Resource Identifiers</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p id="iris-resources">All <a href="#resources">resources</a> provided by a Solid <a href="#Server">server</a> are identified by an <a href="#iri">IRI</a> with the <code>http</code> or <code>https</code> scheme, in the following normal form:</p>
+
+                  <ul id="iris-norm">
+                    <li id="iris-nfc">The IRI is a Unicode string in Normalization Form C (NFC) [<cite><a class="bibref" href="#bib-uax15">UAX15</a></cite>].</li>
+
+                    <li id="#iris-unreserved">The IRI does not contain percent-encoding triplets corresponding to <em>unreserved</em> characters.</li>
+                  
+                    <li id="iris-hex">Hexadecimal digits within percent-encoding triplets corresponding to <em>reserved</em> characters are represented using <em>uppercase</em> letters.</li> 
+
+                    <li id="iris-scheme">The <code>http</code> or <code>https</code> scheme of the IRI is represented using <em>lowercase</em> characters.</li>
+
+                    <li id="#iris-host">The host of the IRI is represented using <em>lowercase</em> characters.</li>
+
+                    <li id="iris-port">If the port of the IRI is the default port for its scheme, the port subcomponent is left out.</li>
+
+                    <li id="iris-path">The path of the IRI does not contain dot-segments.</li>
+                  </ul>
```

Comment:
What do you think about leaving this out? Would conforming to 3987's sections 3.1-3.2 suffice?

```suggestion
```